jiant/proj/main/export_model.py

```diff
@@ -11,30 +12,57 @@
 class RunConfiguration(zconf.RunConfig):
     model_type = zconf.attr(type=str)
     output_base_path = zconf.attr(type=str)
+    hf_model_name = zconf.attr(type=str, default=None)
 
 
-def lookup_and_export_model(model_type, output_base_path):
+def lookup_and_export_model(model_type: str, output_base_path: str, hf_model_name: str = None):
     model_class, tokenizer_class = get_model_and_tokenizer_classes(model_type)
     export_model(
         model_type=model_type,
         output_base_path=output_base_path,
         model_class=model_class,
         tokenizer_class=tokenizer_class,
+        hf_model_name=hf_model_name,
```

Comment:
anything preventing us from aligning `hf_model_name` and `model_type`?

Reply:
Ah, should have mentioned this. `hf_model_name` can contain `/` (e.g. `facebook/bart-large`), whereas we often use `model_type` in paths.

Reply:
If that's the case, I think we should lookup `hf_model_name` in a dict as needed